src/unix/linux_like/linux/uclibc/mips/mod.rs

```diff
@@ -49,6 +49,7 @@ pub const RLIMIT_RSS: ::c_int = 7;
 pub const RLIMIT_NPROC: ::c_int = 8;
 pub const RLIMIT_MEMLOCK: ::c_int = 9;
 pub const RLIMIT_NLIMITS: ::c_int = 15;
+pub const RLIM_NLIMITS: ::c_int = RLIMIT_NLIMITS;
```

Comment:
Same here, I don't think this is MIPS only.

Reply:
I didn't separate it, I just put it next to the existing `RLIMIT_NLIMITS` constant to be safe. At first I was going to put it in `linux_like/mod.rs`, but I could probably compromise and put it in the root of each env submodule.

Reply:
Oh, I see, on uclibc it's only in mips. Should I extract it out to `uclibc/mod.rs`, `musl/mod.rs`, etc?

Reply:
Hm, okay, we should unify it (them) but it isn't necessary on this PR.